.travis.yml

```diff
@@ -1,14 +1,12 @@
-dist: trusty
+dist: xenial
 sudo: false
 language: java
 matrix:
   include:
-  - jdk: oraclejdk8
-    env: TRIPLEA_RELEASE=true
   - jdk: openjdk11
-    env: TRIPLEA_RELEASE=false
+    env: TRIPLEA_RELEASE=true
 addons:
-  postgresql: "9.5"
+  postgresql: "10"
```

Comment:
postgres 10 needed for "offset" (paging) query: https://github.com/triplea-game/triplea/pull/4872/files#diff-0886cbe4926f9b7699f3155e549d7d5eR86